v_4_0_0/master_template.go

```diff
@@ -2057,21 +97,15 @@ coreos:
       ExecStart=/usr/bin/docker run --rm --net=host -v /etc:/etc --name $NAME $IMAGE
       ExecStop=-/usr/bin/docker stop -t 10 $NAME
       ExecStopPost=-/usr/bin/docker rm -f $NAME
```

Comment:
lots of units dont have 
```
      [Install]
      WantedBy=multi-user.target
```
Which is not needed with cloudconfig but will be necessary for ignition